Code/Framework/AzCore/AzCore/Component/Entity.cpp

```diff
@@ -267,11 +270,15 @@ namespace AZ
         m_isDependencyReady = false;
     }
 
+    //! O3DE_DEPRECATION_NOTICE(GHI-19319) - 2025/10/24
+    //! @deprecated use SetStartActive instead.
     void Entity::SetRuntimeActiveByDefault(bool activeByDefault)
     {
         m_isRuntimeActiveByDefault = activeByDefault;
     }
 
+    //! O3DE_DEPRECATION_NOTICE(GHI-19319) - 2025/10/24
+    //! @deprecated use GetStartActive instead.
     bool Entity::IsRuntimeActiveByDefault() const
     {
         return m_isRuntimeActiveByDefault;
```

Comment:
Can we maybe pipe this into the new system somehow instead of keeping the internal bool?